projects/ADuCM360_demo_cn0411/src/CN0411.c

```diff
@@ -1272,8 +1282,12 @@ void CN0411_cmd_cell_const(uint8_t *args, struct cn0411_device *cn0411_dev)
 void CN0411_cmd_solution(uint8_t *args, struct cn0411_device *cn0411_dev)
 {
 	uint8_t *p = args;
+	uint8_t index;
+	float input_val;
+	char *str;
```

Comment:
nitpick: you can probably fit the variables of the same type on one line: `char *str, arg[5];`